fixtures/jira.yaml

```diff
@@ -5,10 +5,9 @@ snyk:
     maturityFilter: proof-of-concept
     type: vuln # what snyk issue types should we consider to ticket
     priorityScoreThreshold: 20
-    remoteUrl: github.com/owner/repo # URL to the remote repository (when not detected)
```

Comment:
why did this go?

Reply:
because it not supported. We were planning to add this and plans have changed so we didn't